api/namex/services/name_request/utils.py

```diff
@@ -123,7 +123,7 @@ def check_ownership(nrd, user):
 def get_or_create_user_by_jwt(jwt_oidc_token):
     # GET existing or CREATE new user based on the JWT info
     try:
-        user = User.find_by_jwtToken(jwt_oidc_token)
```

Comment:
This lookup is made in a few other places.
Will you be changing those as well?
I'm not sure if the tests still work for this, but if they do, it would be good to update them as well